### ISSUE TITLE
Unsure where the new Docker tags are coming from

### ISSUE THREAD

**Issue by Coconutcoo** [DrPsychick/docker-dnsmasq#33]
Hey there! 

Thank you for providing this excellent containerised dnsmasq, I've tried quite a few of the alternatives and yours was the only one that met my needs.

I've noticed that on dockerhub we have newer releases (up to 2.90), but I can't find any reference to that in the source here? The releases on GH stopped back on 2.78, which is fine, perhaps you just don't use that anymore, but I can't find any recent change in any of the source code to know what changed going into 2.90.

Have I just missed a link to a newer repo or something?

**Comment by Coconutcoo** [comment]
Ah, finally reviewed the build logs and realised you just have a cron building it once a week, and it uses whatever the latest dnsmasq at the time - so we may end up overwriting 2.90 for months until 2.91 is released.

**Comment by DrPsychick** [comment]
Indeed. I also did find good alternatives, but I also found many that were "pushed 2 years ago" and that is the reason I built a setup that regularly builds new images to keep them up2date automatically.

The circleci pipeline runs for specific alpine versions and latest regularly and is not pinned to a specific dnsmasq version. `alpine_version: ["latest", "3", "3.18", "3.19", "3.20", "edge"]`

That said, if you're interested in putting in the work to build images for specific dnsmasq version, feel free to submit a PR, I will make sure to review it.
